File: conversions/csharp_unity/Law.cs
using System;
using System.Collections.Generic;
[System.Serializable]
public class Law : BaseElement
{
    // Code
    public string Decree;
    public int? Date;
    public string Purpose;
    public string AuthorId;
    // Compulsion
    public List<string> JurisdictionsIds; = new List<string>();
    public List<string> ProhibitionsIds; = new List<string>();
    public List<string> PenaltiesIds; = new List<string>();
    public List<string> AdjudicatorsIds; = new List<string>();
    public List<string> EnforcersIds; = new List<string>();
}

File: languages/typescript/Law.ts
import { BaseElement } from './base_elements';

export interface Law extends BaseElement {
  // Code
  decree?: string | null;
  date?: number | null;
  purpose?: string | null;
  authorId?: string | null;
  // Compulsion
  jurisdictionsIds?: string[] | null;
  prohibitionsIds?: string[] | null;
  penaltiesIds?: string[] | null;
  adjudicatorsIds?: string[] | null;
  enforcersIds?: string[] | null;
}
